docs/src/main/sphinx/release/release-448.md

```diff
@@ -0,0 +1,45 @@
+# Release 448 (15 May 2024)
+
+## General
+
+* Fix query failure when reading columns with names that contain only
+  non-alphanumeric characters. ({issue}`21833`)
+* Fix potential incorrect results for queries with complex predicates. ({issue}`21887`)
+* Fix potential loss of a failed query when multiple queries are failing at the
```

Comment:
It's a loss of a QueryCompletion event, not of a failed query. Also, "when multiple queries *fail* at the same time", instead of "are failing".